### PR TITLE
Move Code of Conduct into documentation

### PR DESCRIPTION
**Pull request by rrjbca**
## Description
Moves our code of conduct from the top level of the repository into the documentation.

Link from front page: https://skypy--405.org.readthedocs.build/en/405
CoC: https://skypy--405.org.readthedocs.build/en/405/project/CODE_OF_CONDUCT.html
## Checklist
- [x] Follow the [Contributor Guidelines](https://github.com/skypyproject/skypy/blob/master/CONTRIBUTING.md)
- [ ] Write unit tests
- [x] Write documentation strings
- [x] Assign someone from your working team to review this pull request
- [x] Assign someone from the infrastructure team to review this pull request

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,100 +1,5 @@
-The SkyPy Code of Conduct
-=========================
```
Comment (Lucia-Fonseca) [inline CODE_OF_CONDUCT.md:2]: Do we need to get rid of the heading?

**Review by Lucia-Fonseca — COMMENTED** [review]
This pull request does what intends.
The documentation renders well and all links are working.

Do we need a link to the Carpentries Code of Conduct?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (rrjbca) [reply CODE_OF_CONDUCT.md:2]: We should get rid of it, this document is *not* the code of conduct, just a place with a link to the *actual* code of conduct

**Comment by ntessore** [comment]
would much rather have these docs in a separate `skypyproject/policies` or something repo than in the documentation, which should document the code and not our project membership policies

**Review by Lucia-Fonseca — APPROVED** [review]
This works and the documentation renders well. The links also work.

**Pull request merged**